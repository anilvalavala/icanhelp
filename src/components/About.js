import React from 'react';

const About = () => {
    return(
        <div>
            <h1>What is this about?</h1>
            <br />
            <p>
                Yes, today the world is full of sophisticated computers, machines, devices, food, what not, everything. <br />
                But still there are a section of people who are looking for help. <br /><br />
                There are many people who want to extend their helping hand to the society <br />
                But there is no bridge connecting such kind hearted people to the needy ones. <br /> <br />

                So, this application is intended for those people who want to donate something. <br />
                One can simply update their donation details and everyone on web can see it,  
                especially NGOs which are working for upliftment of less privileged. <br />

                This is just a basic verion of my idea. 
                Will keep updating this app to make it more user friendly.
            </p>
        </div>
    );
}

export default About;