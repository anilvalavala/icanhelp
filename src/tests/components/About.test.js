import React from 'react';
import { shallow } from 'enzyme';
import About from '../../components/About';

test('Test About component', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
});