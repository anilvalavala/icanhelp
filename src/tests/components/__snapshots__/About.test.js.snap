// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test About component 1`] = `
<div>
  <h1>
    What is this about?
  </h1>
  <br />
  <p>
    Yes, today the world is full of sophisticated machines, devices, food what not everything. But still there are a section of people who are still looking for help. The needs of those people can only be fulfilled by humanity. There are many people who wants to extend their helping hand to the society, but there is no bridge connecting such kind hearted people to the needy ones. So, this application is intended for those people who want to donate something. One can simply update their donation details and everyone on web can see it, especially NGOs which are working for upliftment of less privileged. This is just a basic verion of my idea. Will keep updating this app to make it more user friendly.
  </p>
</div>
`;
